source/index.html

```diff
@@ -1,12 +1,208 @@
 <!DOCTYPE html>
-<html lang="ru">
+<html class="page" lang="ru">
   <head>
-    <meta charset="utf-8">
+    <meta charset="utf-8" />
     <title>HTML Academy: Седона</title>
   </head>
-  <body>
-
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/adaptive">HTML и CSS. Адаптивная вёрстка и автоматизация</a>».</p>
-
+  <body class="page__body">
+    <!-- Header  -->
+    <header class="page-header">
+      <a class="page-header__logo logo">
+        <img
+          src="img/logotype-mobile.svg"
+          width="102"
+          height="84"
+          alt="Логотип сайта Седона."
+        />
+      </a>
+      <nav class="page-nav">
+        <button class="page-nav__toggle toggle" type="button">
+          <span class="visually-hidden">Открыть меню</span>
+        </button>
+        <!-- <div class="page-nav__wrapper"> -->
+        <ul class="wrapper page-nav__site-list site-list">
```

Comment:
```suggestion
        <ul class="page-nav__site-list site-list">
```
это обсудили. давай без общего врапера (убрать везде нужно)

Reply:
убрала